ext/greenmat/autolink.c

```diff
@@ -223,7 +223,7 @@ sd_autolink__email(
 		return 0;
 
 	for (link_end = 0; link_end < size; ++link_end) {
-		uint8_t c = data[link_end];
+		char c = data[link_end];
```

Comment:
> I understand this change forces the c to be virtually 7-bit ASCII

No, it might be a part of UTF-8's middle bytes, and the change of the type doesn't change the bit pattern.

Because `c` is dealt as `char`, it should be `char`, not `uint8_t` (i.e. `unsigned char`). I believe this code is the best.